src/condor_daemon_client/daemon.cpp

```diff
@@ -1984,6 +1986,24 @@ Daemon::getInfoFromAd( const ClassAd* ad )
 
 	initStringFromAd( ad, ATTR_PLATFORM, &_platform );
 
+	std::string capability;
+	if (ad->EvaluateAttrString(ATTR_REMOTE_ADMIN_CAP, capability)) {
+		ClaimIdParser cidp(capability.c_str());
+		dprintf(D_FULLDEBUG, "Creating a new administrative session for capability %s\n", capability.c_str());
```

Comment:
This dprintf() leaks the secret part of the capability. Use ClaimIdParser::publicClaimId() to print just the non-secret portion.